client/src/components/Table/Table.js

```diff
@@ -54,6 +57,10 @@ export default function CustomTable(props) {
               return (
                 <TableRow key={k} className={classes.tableBodyRow}>
                   {prop.map((prop, key) => {
```

Comment:
same name of array as well element. increases confusion.

Reply:
This is Material UI code. We can work on it separately.